### DISCUSSION TITLE
Welcome to summer-school-2022 Discussions!

### DISCUSSION THREAD

**Discussion by stephank16** [IS-ENES-Data/summer-school-2022#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the IS-ENES summer school. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Use with team members to explore topics💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Marco-DKRZ** [comment]
Hi I'm Marco and this is a test comment.
